tracer/test/Datadog.Trace.IntegrationTests/SpanTagTests.cs

```diff
@@ -26,182 +28,193 @@ public SpanTagTests()
             var matchAllRule = "[{\"service\":\"*\", \"name\":\"*\", \"sample_rate\":1.0, \"max_per_second\":1000.0}]";
             var settings = TracerSettings.Create(new() { { ConfigurationKeys.SpanSamplingRules, matchAllRule } });
             _writer = new AgentWriter(_testApi, statsAggregator: null, statsd: null);
-            _tracer = new Tracer(settings, _writer, sampler: null, scopeManager: null, statsd: null);
+            _tracer = ScopedTracerHelper.BuildScopedTracer(settings, _writer, sampler: null, scopeManager: null, statsd: null);
```

Comment:
nit: this would all be cleaner if we stopped having "field" tracer instances and used local variables instead, so you can do `await using var tracer = ScopedTracerHelper.BuildScopedTracer()` like you do elsewhere 😄

Reply:
In fact, I just extracted all your `ScopedTracer` work into a separate PR here #7220 in the hopes we can get that merged sooner rather than later 🙂